tools/webdriver/webdriver/bidi/modules/browsing_context.py

```diff
@@ -35,9 +35,16 @@ def activate(self, context: str) -> Mapping[str, Any]:
 
     @command
     def capture_screenshot(
-        self, context: str, clip: Optional[ClipOptions] = None
-    ) -> Mapping[str, Any]:
-        params: MutableMapping[str, Any] = {"context": context}
+            self,
+            context: str,
+            format: Optional[str] = None,
+            origin: Optional[str] = None,
+            clip: Optional[ClipOptions] = None) -> Mapping[str, Any]:
+        params: MutableMapping[str, Any] = {
```

Comment:
I think we should only set keys if the value is not None. None would serialize as null which is not what we want.